extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeMusicSearchExtractorTest.java

```diff
@@ -130,6 +130,7 @@ public static void setUp() throws Exception {
         @Override public InfoItem.InfoType expectedInfoItemType() { return InfoItem.InfoType.CHANNEL; }
     }
 
+    @Disabled("Currently constantly switching between \"Did you mean\" and \"Showing results for ...\" occurs")
```

Comment:
You could make this `@MockOnly` instead for now, since we still want to test that the code correctly extracts the suggestion from the stored mocks.

Reply:
Useless in my opinion because the tests are based on outdated data when actually testing it, it can still fail and the test says that everything is ok.

Reply:
That's true, but at least the test can be used to verify that code changes inside NewPipeExtractor don't break currently working features.

Reply:
Note: ``YoutubeMusicSearchExtractorTest$Suggestion`` has no mock data!